services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/DiagnosisKeysStructureProvider.java

```diff
@@ -51,6 +51,9 @@
   public Directory<WritableOnDisk> getDiagnosisKeys() {
     logger.debug("Querying diagnosis keys from the database...");
     Collection<DiagnosisKey> diagnosisKeys = diagnosisKeyService.getDiagnosisKeys();
+
+    diagnosisKeys.removeIf(key -> key.getTransmissionRiskLevel() < 3);
```

Comment:
@corona-warn-app/cwa-server-codeowners make the '3' a configurable environment parameter